libs/core/src/lib/select/select.component.html

```diff
@@ -1,90 +1,103 @@
-<ng-container *ngTemplateOutlet="mobile ? selectMobileTemplate : selectDesktopTemplate"></ng-container>
+<ng-container *ngTemplateOutlet="mobile ? mobileTemplate : desktopTemplate"></ng-container>
 
-<ng-template #selectDesktopTemplate>
-    <fd-popover
-        additionalBodyClass="fd-popover-custom-list"
-        [isOpen]="isOpen"
-        [triggers]="[]"
-        [focusTrapped]="true"
-        [disabled]="disabled"
-        [appendTo]="appendTo"
-        [options]="popperOptions"
-        [closeOnEscapeKey]="false"
-        [fillControlMode]="'at-least'"
-        [maxWidth]="600"
-        [closeOnOutsideClick]="closeOnOutsideClick"
-        (isOpenChange)="popoverOpenChangeHandle($event)">
+<!--
+    With CDK overlays the origin trigger and connected overlay needs to be together within one level
```

Comment:
Please consider using Popover that is overlay inside. I am not sure what performance impact we will get and we need to test it.